rocAL/rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -593,62 +511,22 @@ extern "C"  RocalImage  ROCAL_API_CALL rocalVideoFileSourceSingleShard(RocalCont
                                                                     unsigned stride = 0,
                                                                     bool file_list_frame_num = true);
 
-/// Creates a video reader and decoder as a source. It allocates the resources and objects required to read and decode mp4 videos stored on the file systems. Resizes the decoded frames to the dest width and height.
+/// Creates a video reader and decoder as a source. It allocates the resources and objects required to read and decode mp4 videos stored on the file systems.
 /// \param context Rocal context
 /// \param source_path A NULL terminated char string pointing to the location on the disk.
 /// source_path can be a video file, folder containing videos or a text file
 /// \param color_format The color format the frames will be decoded to.
 /// \param rocal_decode_device Enables software or hardware decoding. Currently only software decoding is supported.
 /// \param internal_shard_count Defines the parallelism level by internally sharding the input dataset and load/decode using multiple decoder/loader instances.
 /// \param sequence_length: The number of frames in a sequence.
```

Comment:
Either remove all video API's. Or just keep the API header declaration.

Reply:
Done